src/main/java/picard/analysis/TheoreticalSensitivityMetrics.java

```diff
@@ -0,0 +1,33 @@
+package picard.analysis;
+
+import htsjdk.samtools.metrics.MetricBase;
+
+/**
+ * Created by fleharty on 6/30/17.
+ */
+public class TheoreticalSensitivityMetrics extends MetricBase {
```

Comment:
The "key" to the histogram is of any Comparable type. Integer is Comparable, so that works, but you can use Double as well. You could use a String even! You can see an example for having a double in the histogram here: 
picard/sam/markduplicates/util/AbstractMarkDuplicatesCommandLineProgram.java:162